src/cargo/core/dependency.rs

```diff
@@ -54,6 +67,16 @@ pub enum Kind {
     Build,
 }
 
+impl Encodable for Kind {
+    fn encode<S: Encoder>(&self, s: &mut S) -> Result<(), S::Error> {
+        match *self {
+            Kind::Normal => "normal",
+            Kind::Development => "development",
```

Comment:
This may want to just be called "dev" as that's what it's called in the manifest